pyigm/abssys/lls.py

```diff
@@ -498,6 +498,7 @@ def tau_multi_lls(wave, all_lls, **kwargs):
     from linetools.analysis import voigt as lav
     #
     all_tau_model = np.zeros(len(wave))
+    all_tau_model = list(np.zeros(len(wave)))  ## need list not array for python3 ?
```

Comment:
Please explain why this needs to be a list.

Reply:
There is no way that 

```
all_tau_model = np.zeros(len(wave))
```

will give a _set_unit error.

You must be referring to other code.  
Please expplain.

Reply:
The part of the code that is giving that error is
(line 519 in lls.py)   all_tau_model += tau_model

Reply:
Then why make the change here?  Might it not break

Reply:
I would propose making the change in the same place, and adding at the end of the tau_multi_lls function something that would convert tau_multi_lls to its original type (analogue to all_tau_model = np.asarray(all_tau_model)). Could you let me know what is the original array type? Thanks.

Reply:
I tried adding all_tau_model = np.asarray(all_tau_model) at the end, but the gui was then not able to fit LLS. Also redshift for continuum fitting could not be QSO redshift. I suggest we discuss this more tomorrow, and if you can show me an example how is the gui supposed to work. I would also suggest adding docs on this gui.

Reply:
Please do add docs on the GUI.  I'm making an Issue
and assigning it to you.